### DISCUSSION TITLE
AlpineJS not reinitialised from using Stimulus Reflex

### DISCUSSION THREAD

**Discussion by acetinick** [alpinejs/alpine#823] [4. General]
We are using Rails library [Stimulus Reflex](https://github.com/hopsoft/stimulus_reflex),  across our views, and found that AlpineJS components that have been inserted after reflex updates were not reinitialised properly.

I noticed a similar issue discused [here ](https://github.com/phoenixframework/phoenix_live_view/issues/809) with LiveView library, and wondering if we could get some help to enable support for Stimulus Reflex.

**Comment by juddkirby** [comment]
Stimulus Reflex now contains an example detailing how to support AlpineJS, it's located at the end of the [Customization -> shouldMorph section](https://cableready.stimulusreflex.com/customization#shouldmorph-callbacks) of their official docs.  This should provide the solution you need.
